homeassistant/components/aftership/sensor.py

```diff
@@ -58,19 +60,43 @@ async def async_setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the AfterShip sensor platform."""
-    apikey = config[CONF_API_KEY]
-    name = config[CONF_NAME]
```

Comment:
Previously, a user could specify the sensor's name, which is, with this change, not possible anymore.

Reply:
Yes, the old name is transferred to the config entry title. In the future people can rename the device/config entry to rename their stuff (we are slowly migrating away from settable names in configurations)